composer.json

```diff
@@ -15,17 +15,21 @@
         "symfony-cmf/routing-auto": "^2.0",
         "symfony-cmf/routing-bundle": "^2.0",
         "aferrandini/urlizer": "1.0.*",
-        "phpdocumentor/reflection-docblock": "^3.1, !=3.2.0"
+        "phpdocumentor/reflection-docblock": "^3.1, !=3.2.0",
+        "symfony-cmf/core-bundle": "^2.0"
```

Comment:
somehow this conversation disapeared. anyways, just checked and you don't need the interface existing to do the check. see https://3v4l.org/9E09Q - when the interface does not exist, the `instanceof` will just always be false. as long as you provide a model without the interface, things should work out fine. here we seem to only do that for tests, so all should be well.

Reply:
Good to know! Thanks!